prefab_build.sh

```diff
@@ -0,0 +1,88 @@
+# Test that the prefab binary exists
+if hash prefab 2>/dev/null; then
+  echo "Prefab is installed"
+else
+  echo "Prefab binary not found. See https://github.com/google/prefab for install instructions"
+  exit 1
+fi
+
+# Get the version string from the source
+major=`grep "#define OBOE_VERSION_MAJOR" include/oboe/Version.h | cut -d' ' -f3`
```

Comment:
Nice!
BTW, Mikhail says the back tick is obsolete and recommends:

    major=$(grep "#define OBOE_VERSION_MAJOR" include/oboe/Version.h | cut -d' ' -f3)

Reply:
Good to know - thanks!